@xen-orchestra/lite/src/components/pool/dashboard/cpuUsage/PoolCpuUsageChart.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <LinearChart
+    :title="$t('pool-cpu-usage')"
+    :subtitle="$t('last-week')"
+    :data="data"
+    :value-formatter="customValueFormatter"
+  />
```

Comment:
For CPU usage, the max value of the graph should be fixed and equal to the number of CPUs * 100. e.g.: 200% in the mockup even if it never comes close to 200% usage.

Reply:
It shouldn't be static. 200 only means that there are 2 CPUs. It should depend on the number of CPUs.